metaserver/src/main/scala/scala/meta/languageserver/providers/DocumentHighlightProvider.scala

```diff
@@ -0,0 +1,33 @@
+package scala.meta.languageserver.providers
+
+import com.typesafe.scalalogging.LazyLogging
+import org.langmeta.io.AbsolutePath
+import langserver.{types => l}
+import langserver.messages.DocumentHighlightResult
+import scala.meta.languageserver.Uri
+import scala.meta.languageserver.search.SymbolIndex
+import scala.meta.languageserver.ScalametaEnrichments._
+
+object DocumentHighlightProvider extends LazyLogging {
+
+  def highlight(
+      symbolIndex: SymbolIndex,
+      path: AbsolutePath,
+      position: l.Position
+  ): DocumentHighlightResult = {
+    logger.info(s"Document highlight in ${path}")
+    val locations = for {
+      name <- symbolIndex
+        .resolveName(path, position.line, position.character)
+        .toList
+      data <- symbolIndex.symbolIndexer.get(name.symbol).toList
+      _ = logger.info(s"Highlighting symbol `${data.name}: ${data.signature}`")
```

Comment:
debug?

Reply:
I'd leave it `info` for now, it's only once per call (note that `Found highlight` is `debug`). Tell me if you still want me to change it, I don't mind.